applications/matter_weather_station/src/app_task.cpp

```diff
@@ -208,7 +231,7 @@ int AppTask::StartApp()
 	while (true) {
 		k_msgq_get(&sAppEventQueue, &event, K_FOREVER);
 		DispatchEvent(event);
-	}
+		}
```

Comment:
Is everything with formatting ok? I've seen similar change below and it looks for me like some bracket was removed by accident.

Reply:
Right, I'll reformat the file.